x-pack/plugins/infra/server/graphql/metadata/schema.gql.ts

```diff
@@ -6,15 +6,15 @@
 
 import gql from 'graphql-tag';
 
-export const capabilitiesSchema = gql`
-  "One specific capability available on a node. A capability corresponds to a fileset or metricset"
-  type InfraNodeCapability {
+export const metadataSchema = gql`
+  "One specific metadata information for a node."
```

Comment:
There is something about information always being plural in english. How about "A metadata entry for a specific node" to circumvent that?

Reply:
That's better, thank you.